dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/ExistingAccountEmailConfirmation.cshtml

```diff
@@ -15,9 +15,13 @@
 
             <p>A confirmation code has been sent to <b data-testid="email">@Redactor.RedactEmail(Model.Email!)</b></p>
 
-            <govuk-input asp-for="Code" input-class="govuk-!-width-one-quarter" pattern="[0-9]*" inputmode="numeric" autocomplete="one-time-code">
-                <govuk-input-label class="govuk-label--s" />
-            </govuk-input>
+            <govuk-input
+                asp-for="Code"
+                input-class="govuk-!-width-one-quarter"
```

Comment:
does this also need `govuk-input--extra-letter-spacing` ?

Reply:
it does